DEPLOYING.md

```diff
@@ -0,0 +1,32 @@
+How to Deploy to Maven Central (for Maintainers Only)
+-----------------------------------------------------
+
+If you haven't deployed artifacts to Maven Central before:
+- Follow the instructions on [this
+  page](http://central.sonatype.org/pages/ossrh-guide.html) to set up an
+  account with OSSRH.
+- Install GnuPG and [generate your key pair](https://www.gnupg.org/documentation/howtos.html)
+- Put your GnuPG key password and OSSRH account information in
```

Comment:
You actually don't need GnuPG to make this work.  You can just use a username/password.

@ejona86  thoughts?